### ISSUE TITLE
Fragments generate incorrect "import" and "package" information

### ISSUE THREAD

**Issue by jtwigg** [apollostack/apollo-android#107]
given the following modification to the android/apollo-sample project
```
query DroidDetails {
  species(id: "c3BlY2llczoy") {
    id
    name
    ...SpeciesInformation

  }
}

fragment SpeciesInformation on Species{
  classification
}
```

if generates an extra **dot** for the _import_ and _package_ statements

```
//DroidDetails.java
import .fragment.SpeciesInformation;
```

```
//fragments/SpeciesInformat.java
package .fragment;
```

**Comment by martijnwalraven** [comment]
@ivans: Could this have something to do with a missing `filePath` in the `apollo-codegen` output?

The Android compiler relies on some recently added information about the file path of `.graphql` files, so for now it has to be installed with `npm install -g apollo-codegen@next`. Not sure how the Gradle plugin does the installation.

@jtwigg: Could you check what version of `apollo-codegen` you have installed? (`apollo-codegen --version`)

**Comment by jtwigg** [comment]
Version : 0.9.5
Shows error.

will try updating...

Version : 0.10.1
Happens as well.

**Comment by marwanad** [comment]
@martijnwalraven I think it's not related to `apollo-codegen` because the plugin uses a locally installed one.

It's because the schema file is under `src/main/graphql` so when we format the package name it's an empty string. We're always doing `irPackageName.fragment` but it should be something like
`irPackageName? irPackageName.fragment: irPackageName`

**Comment by jtwigg** [comment]
@marwanad CAn you suggest and alternative format for my query that you believe will pass? I'd like to see if I can get things further along.

**Comment by marwanad** [comment]
@jtwigg yup try having the schema.json file under something like`src/main/graphql/api` for example.

**Comment by jtwigg** [comment]
@marwanad It improves the results, DroidDetails now compiles, but Film.java does not because it unnecessarily tries to reference the Fragment Schema without importing the package.

```
//Film.java

  public static final String QUERY_DOCUMENT = OPERATION_DEFINITION + "\n"
   + SpeciesInformation.FRAGMENT_DEFINITION;
//Error:(24, 6) error: cannot find symbol variable SpeciesInformation
```

**Comment by marwanad** [comment]
@jtwigg found the cause. It should be fixed with #108. 
I added your query file there under the test fixtures. Thanks for reporting!
